### ISSUE TITLE
[CI] EqlSpecIT test {sequenceManyManyCondition} failing

### ISSUE THREAD

**Issue by andreidan** [elastic/elasticsearch#74685]
**Build scan**: https://gradle-enterprise.elastic.co/s/jjjirfq4kjnca

**Repro line**:
```
 ./gradlew ':x-pack:plugin:eql:qa:multi-cluster-with-security:integTest' --tests "org.elasticsearch.xpack.eql.EqlSpecIT.test {sequenceManyManyCondition}" \
  -Dtests.seed=4AB4B88F796A5392 \
  -Dtests.locale=is-IS \
  -Dtests.timezone=America/Argentina/Tucuman \
  -Druntime.java=8
```

**Reproduces locally?**: no

**Applicable branches**: 7.x, master

**Failure history**:

**Failure excerpt**:
```
java.net.SocketTimeoutException: 10.000 milliseconds timeout on connection http-outgoing-2 [ACTIVE]
	at __randomizedtesting.SeedInfo.seed([4AB4B88F796A5392:C2E08755D7963E6A]:0)
	at org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:866)
	at org.elasticsearch.client.RestClient.performRequest(RestClient.java:283)
	at org.elasticsearch.client.RestClient.performRequest(RestClient.java:286)
	at org.elasticsearch.client.RestClient.performRequest(RestClient.java:270)
	at org.elasticsearch.client.RestHighLevelClient.performClientRequest(RestHighLevelClient.java:2082)
	at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1732)
	at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1717)
	at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1684)
	at org.elasticsearch.client.EqlClient.search(EqlClient.java:47)
	at org.elasticsearch.test.eql.BaseEqlSpecTestCase.runRequest(BaseEqlSpecTestCase.java:135)
	at org.elasticsearch.test.eql.BaseEqlSpecTestCase.runQuery(BaseEqlSpecTestCase.java:124)
	at org.elasticsearch.test.eql.BaseEqlSpecTestCase.test(BaseEqlSpecTestCase.java:96)
```

**Comment by bpintea** [comment]
This issue doesn't seem to have an obvious test-related trigger, but might potentially be alleviated once #74815 is fixed.

**Comment by bpintea** [comment]
It seems that some CCS requests simply take longer than 10s to get a response in CI. I guess we'll just need to up the timeout, at least for now.

**Comment by mayya-sharipova** [comment]
Another [failure](https://gradle-enterprise.elastic.co/s/rlkwmyr7km3au) on PR on master.

Looks to be related to timeouts.

**Comment by cbuescher** [comment]
Some more like this today on 7.x: https://gradle-enterprise.elastic.co/s/oydm7i36jfiyq
@bpintea only adding that build here for awareness, you can probably better judge if the timeouts you already increased need more adaption or not.

**Comment by ywangd** [comment]
Failed again on 7.14. I am reopenning it since the fix should have been backported to 7.14 but didn't seem to fully work.

**Comment by bpintea** [comment]
The backporting PRs for 7.14 and 7.x had been [auto] opened but not [auto] merged (missing label). Did that now, hopefully this failure will no longer happen.
